src/main/java/uk/gov/hmcts/reform/pip/data/management/utils/JsonExtractor.java

```diff
@@ -53,10 +67,28 @@ public JsonExtractor(SearchConfiguration searchConfiguration) {
     @Override
     public boolean isAccepted(String payload) {
         try {
+            // test if the file is json format
             new ObjectMapper().readTree(payload);
+            // validate json file with JSON schema
             return true;
         } catch (IOException exception) {
             return false;
         }
     }
+
+    @Override
+    public List<String> validate(String payload) {
```

Comment:
Awesome :) Really like this method, nice and elegant

Reply:
thank you